### PR TITLE
[issue_tracker] Fixes bug when no issue comment

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

Solves error when issueComment is null.

#### Testing instructions (if applicable)

1. Checkout PR
2. View issue when no comment

#5365

### CODE REVIEW

File: modules/user_accounts/php/edit_user.class.inc [hunk 1]
```diff
@@ -80,7 +80,7 @@ class Edit_User extends \NDB_Form
         $defaults = array();
 
         if (!$this->isCreatingNewUser()) {
-            $user = \User::factory($this->identifier);
+            $user = \User::factory(urldecode($this->identifier));
```
[the remark this answers is not in the document]
Comment (maltheism) [reply modules/user_accounts/php/edit_user.class.inc]: @johnsaigle you're fast. My master branch for some reason had that in it and I removed it just now.

**Comment by johnsaigle** [comment]
@ridz1208 I believe it was originally reported for 21.

**Comment by ridz1208** [comment]
@maltheism if its ready to be tested, please rebase on 21.0-release for now

**Comment by maltheism** [comment]
See https://github.com/aces/Loris/pull/5580

**Pull request closed without merging**